packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -101,7 +101,7 @@ export class ResultUtils {
     selector = undefined
   ) {
     const interactiveResult = controllerBuilder(engine, {
-      options: {result: JSON.parse(JSON.stringify(result))},
+      options: {result: result},
```

Comment:
This probably has more places than just the quickview where it's used? Are you sure it still works everywhere else it was used?

Reply:
yes! the other places where this used were also tested.